### PR TITLE
Added Channel Reserve to Glossary

### PR DESCRIPTION
**Pull request by Mogashni**
Added a short paragraph explanation of channel reserve to glossary.

[Link to live preview](https://deploy-preview-811--sad-borg-390916.netlify.app/)

### CODE REVIEW

**Review by sbddesign — CHANGES_REQUESTED** [review]
Nice! 2 things:
- Need a comma
- Maybe we should specify "lightning payment channel" for someone who's unfamiliar?

For future PRs: one thing that's helpful for people reviewing is if you put the [deploy preview link](https://deploy-preview-811--sad-borg-390916.netlify.app/guide/glossary/#channel-reserve) in the description so they can skip directly to what they are reviewing.

File: guide/glossary/index.md [hunk 1]
```diff
@@ -111,6 +111,10 @@ The described process is part of bitcoins hard-coded monetary policy, which foll
 
 A standardized technical document format for the Lightning network protocol specifications. They are hosted on Github [here](https://github.com/lightningnetwork/lightning-rfc). The various Lightning implementations must adhere to the BOLTs in order to be interoperable. However, some Lightning implementations may have features that are not defined in BOLTs.
 
+### Channel Reserve
+
+A channel reserve works as a type of insurance against theft. If a peer tries to cheat in a channel then the other party can submit a penalty transaction. This transaction will then take away all the funds from the other user's channel. Having the channel reserve in place ensures that there are funds available to take away should this occur.
```
[the remark this answers is not in the document]
Comment (GBKS) [reply guide/glossary/index.md:116]: Good idea to work in the word lightning in this paragraph. Might otherwise not be clear which layer this is about.

**Review by sbddesign — APPROVED** [review]
💯

**Comment by Mogashni** [comment]
Have added a link to the live preview at the top as well. Thanks for your comments!

**Review by GBKS — APPROVED** [review]
ACK. Good addition.

**Pull request merged**